core/hybrisadaptor.cpp

```diff
@@ -212,7 +212,7 @@ void HybrisManager::stopReader(HybrisAdaptor *adaptor)
                 sensordLogW() <<Q_FUNC_INFO<< "failed for"<< strerror(-error);
             }
         }
-        if (list.at(i) != adaptor && list.at(i)->isRunning()) {
+        if (list.at(i) != adaptor && list.at(i)->shouldBeRunning_) {
```

Comment:
Is there actually a way for shouldBeRunning_ to be different from isRunning()? To me it seems every function that modifies running_ or shouldBeRunning_ returns with running_ equal to shouldBeRunning_. Ok, I did not check other source files.